lib/private/Setup.php

```diff
@@ -542,19 +542,27 @@ public static function protectDataDirectory() {
 		//Require all denied
 		$now =  date('Y-m-d H:i:s');
 		$content = "# Generated by Nextcloud on $now\n";
-		$content.= "# line below if for Apache 2.4\n";
+		$content.= "# Section for Apache 2.4 and 2.5\n";
 		$content.= "<ifModule mod_authz_core.c>\n";
-		$content.= "Require all denied\n";
+		$content.= "  Require all denied\n";
+		$content.= "</ifModule>\n";
```

Comment:
```suggestion
		$content .= "</ifModule>\n";
```

There should be a whitespace between $content and `.=`. Mind to change it? I know it was like that before.

Reply:
@kesselb 
Makes sense, the whitespace is also used throughout other code as well as `ifModule` => `IfModule` you suggested below. Lets keep it consistent and stay with syntax according to official docs, regarding cases.